src/codegen/codegen_llvm.cpp

```diff
@@ -622,23 +904,37 @@ void CodeGen_LLVM::visit(const Yield* op) {
 }
 
 void CodeGen_LLVM::visit(const Allocate* op) {
-  auto _ = CodeGen_LLVM::IndentHelper(this, "Allocate");
-
-  auto voidptr = get_void_ptr_type(*this->Context);
-  auto i64 = get_int_type(64, *this->Context);
-
-  auto var = codegen(op->var);
-  auto num_elements = this->Builder->CreateZExt(codegen(op->num_elements), i64);
-  if (op->is_realloc) {
-    auto size = this->Builder->CreateMul(
-        num_elements, get_int_constant(64, 4, *this->Context), "realloc.size");
-    auto ret = emitExternalCall("realloc", voidptr, {voidptr, i64}, {var, size});
-    ret->setName("realloc.ret");
-  } else {
-    auto size = get_int_constant(64, 4, *this->Context);
-    auto ret = emitExternalCall("calloc", voidptr, {i64, i64}, {num_elements, size});
-    ret->setName("calloc.ret");
-  }
+  auto _ = CodeGen_LLVM::IndentHelper(this, "Warning: Missing Allocate implementation");
+  /*
+    auto _ = CodeGen_LLVM::IndentHelper(this, "Allocate");
+    auto voidptr = get_void_ptr_type(*this->Context);
+    auto i64 = get_int_type(64, *this->Context);
+    auto i32p = get_int_ptr_type(32, *this->Context);
+
+    auto alloca = this->Builder->CreateAlloca(i32p);
+  //  auto var = codegen(op->var);
+    auto num_elements = codegen(op->num_elements);
+    auto sext_num_elements = this->Builder->CreateSExt(num_elements, i64);
+
+    if (op->is_realloc) {
+      auto size = this->Builder->CreateMul(
+          sext_num_elements, get_int_constant(64, 4, *this->Context), "realloc.size");
+
+      auto ret = emitExternalCall("realloc", voidptr, {i64}, {size});
+      ret->setName("realloc.ret");
+
+      auto bitcast = this->Builder->CreateBitCast(ret, i32p);
+      this->Builder->CreateStore(bitcast, alloca);
+    } else {
+      auto size = this->Builder->CreateMul(
+        sext_num_elements, get_int_constant(64, 4, *this->Context));
+
+     auto call = emitExternalCall("malloc", voidptr, {i64}, {size});
+     call->setName("calloc.ret");
+
+     auto bitcast = this->Builder->CreateBitCast(call, i32p);
+     this->Builder->CreateStore(bitcast, alloca);
+    }*/
```

Comment:
Let's do this in a later PR. Can you emit a warning here saying we're not emitting code for `Allocate`

Reply:
Should I add a `throw logic_error("Not Implemented for Allocate.");` after `auto _ = CodeGen_LLVM::IndentHelper(this, "Warning: Missing Allocate implementation");` ?

Reply:
No, use `taco_uwarning` for the warning message

```suggestion
 taco_uwarning << "Missing LLVM implementation for Allocate opcode" << std::endl;
```

Reply:
`taco_uwarning` stops the code generation. Are ther any other instruction that issue a warning without interrupting the code generation process?

Reply:
Apply the following diff

```
diff --git a/src/error.cpp b/src/error.cpp
index 9be355db..90292a45 100644
--- a/src/error.cpp
+++ b/src/error.cpp
@@ -62,7 +62,8 @@ ErrorReport::ErrorReport(const char *file, const char *func, int line,
 void ErrorReport::explodeWithException() {
   TacoException e = TacoException(msg->str());
   delete msg;
-  throw e;
+  if (!warning)
+    throw e;
 }

 }
```